pykickstart/sections.py

```diff
@@ -494,6 +494,38 @@ def _getParser(self):
                             with no additional characters (the dhcp line in the
                             example above is an individual package).
 
+                            The ``@`` prefix is also used to request installation
+                            of module streams in the following format::
+
+                                @<module name>:<stream name>/<profile name>
+
+                            Profile name is optional and multiple profiles can be
+                            installed by using multiple lines, one per profile.
+                            Stream name is only optional only if the given module
+                            has a default stream.
+
+                            If there are a module and a group named the same,
+                            and no stream name and profile are specified,
+                            module will be selected instead of a group.
```

Comment:
How do you select the group then? Wouldn't it make more sense to select the group if there is a name conflict, and require the module to have the stream specified?